constants/constants.go

```diff
@@ -3,10 +3,12 @@ package constants
 const (
 	OperatorSecretEnabledEnvVar         = "OPERATOR_SECRET_ENABLED"
 	EnterprisePortalSecretEnabledEnvVar = "ENTERPRISE_PORTAL_SECRET_ENABLED"
```

Comment:
I do not think we should continue maintaining anything related to `EnterprisePortalSecretEnabledEnvVar` old naming of the Developer Portal chart.

Reply:
I mean old charts will use releases with `v1.x.y`, for instance, v1.5.0 where the script already uses old names for Tyk Dev Portal. This change only required in the new charts, which will be released through v2.x.y as you suggested

Reply:
Ok, I see your point now. Adapted.